frontend/ui/data/css_tweaks.lua

```diff
@@ -301,6 +301,14 @@ h1, h2, h3, h4, h5, h6 { hyphens: none !important; }
 * { font-variant: no-common-ligatures; }
                 ]],
             },
+            {
+                id = "ruby_inline";
+                title = _("Render <ruby> content inline"),
+                description = _("Disable handling of <ruby> tags and have them rendered inline."),
```

Comment:
I just realized "have them rendered inline" seems a bit wordy. Maybe like this?
```suggestion
                description = _("Disable handling of <ruby> tags and render them inline."),
```